client.go

```diff
@@ -1326,79 +1292,48 @@ func (c *Client) refreshCmd(cmd *protocol.RefreshRequest) (*clientproto.RefreshR
 
 	resp := &clientproto.RefreshResponse{}
 
-	token := cmd.Token
-	if token == "" {
+	if cmd.Token == "" {
 		c.node.logger.log(newLogEntry(LogLevelInfo, "client token required to refresh", map[string]interface{}{"user": c.user, "client": c.uid}))
 		return resp, DisconnectBadRequest
 	}
 
 	config := c.node.Config()
 
-	var user string
-	var expireAt int64
-	var info Raw
-	var b64info string
-
-	if cmd.Token == "" {
```

Comment:
looks like previously we had duplicate checks here with different return disconnect codes:) Thanks for fixing.